smithy-aws-protocol-tests/model/restJson1/http-content-type.smithy

```diff
@@ -286,25 +333,46 @@ structure TestNoPayloadInputOutput {
 /// This example operation has no input and serializes a request without an HTTP body.
 ///
 /// These tests are to ensure we do not attach a body or related headers
-/// (Content-Length, Content-Type) to operations that semantically
+/// (Content-, Content-Type) to operations that semantically
 /// cannot produce an HTTP body.
 ///
 @readonly
 @http(uri: "/no_input_no_payload", method: "GET")
-operation TestNoInputNoPayload {
+operation TestGetNoInputNoPayload {
+    output: TestNoPayloadInputOutput
+}
+
+@readonly
```

Comment:
I think it's better to put the operation and its tests in order.

i.e., 
/// docs about TestGetNoInputNoPayload
TestGetNoInputNoPayload
tests for TestGetNoInputNoPayload
/// docs about TestPostNoInputNoPayload
TestPostNoInputNoPayload
tests for TestPostNoInputNoPayload